content/en/docs/concepts/overview/object-management-kubectl/kustomization.md

```diff
@@ -163,6 +170,7 @@ metadata:
 ### Setting cross-cutting fields
 It is quite common to set some cross-cutting fields at the same time for all
 Kubernetes Resources that are in the same project
+
```

Comment:
nit: do you need `at the same time`?
Suggested edit: 
It is quite common to set cross-cutting fields for all Kubernetes resources in a project. Some use cases for setting cross-cutting fields:

Reply:
updated